packages/ui-components/jest.testing.config.js

```diff
@@ -5,9 +5,7 @@ module.exports = {
     "\\.(css|scss)$": "identity-obj-proxy",
   },
   roots: ["<rootDir>/src"],
-  setupFilesAfterEnv: [
-    "@testing-library/jest-dom/extend-expect",
-  ],
+  setupFilesAfterEnv: ["@testing-library/jest-dom/extend-expect"],
```

Comment:
nit: This file could be excluded from this PR since this diff is from previous changes that were reverted.